docs/examples/tools/cloud_resource_detector/README.rst

```diff
@@ -44,3 +44,50 @@ After running the metrics example:
     * You can filter by resource info and change the graphical output here as well.
 
 Or, if you ran the tracing example, you can go to `Cloud Trace overview <https://console.cloud.google.com/traces/list>`_ to see the results.
+
+
+Troubleshooting
+--------------------------
+
+gke_container resources are not being detected or exported:
+###########################################################
+You need to manually pass in some information via the `Downward API <https://kubernetes.io/docs/tasks/inject-data-application/environment-variable-expose-pod-information/>`_
+to enable GKE resource detection. Your kubernetes config file should look
+something like this (passing in NAMESPACE, CONTAINER_NAME, POD_NAME)
+
+.. code-block:: sh
```

Comment:
should be yaml or yml? Not sure which one
```suggestion
.. code-block:: yml
```

Reply:
After trying both `yml` and `yaml`, it seems they are the same, except `yaml` has colors. Going with the colorful one :)